hack/lib/common.sh

```diff
@@ -31,3 +31,7 @@ function is_installed {
   fi
   return 1
 }
+
+function install_service_monitor_crd {
+  kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/3d6df15f3ac9ce060f9d3d4ba6db7aed5f706ea0/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
```

Comment:
```suggestion
  // It will install the monitor service CRD from the release-0.35 
  monitoServiceCRDUrl := https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.35/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
  kubectl apply -f monitoServiceCRDUrl
```